src/helpers/__tests__/AuthHelper.test.ts

```diff
@@ -0,0 +1,5 @@
+test('dummy test', () => {
+  expect(1).toBe(1)
+})
+
+export {}
```

Comment:
once we have an import statement we can drop this, no fix is needed, so just keep here for a while